storage/pipeline/fsm.go

```diff
@@ -796,3 +825,41 @@ func planOneOrIgnore(ts ...func() (mut mutator, next func(*SectorInfo) (more boo
 		return cnt, nil
 	}
 }
+
+func maybeNotifyRemoteDone(success bool, state string) func(*SectorInfo) {
```

Comment:
why "maybe"? I think that naming becomes confusing as to the intent here

Reply:
`bestEffortNotifyRemoteDone` or something similar might be more clear?

Reply:
This `maybe` tries to refer to some FP concepts around optionality, which in this context tries to relay that an action will be performed if some condition is met (the SealingDone endpoint is set)

Reply:
IMO, I think it would be better if we properly document what that optionality means i.e what conditions need to be met for the action to be performed and just call it NotifyRemoteDone.